addons/project/data/mail_template_data.xml

```diff
@@ -57,26 +57,26 @@
             </t>
         </td></tr>
         <tr><td style="text-align: center;">
-            <table border="0" cellpadding="0" cellspacing="0" width="590" summary="o_mail_notification" style="width:100%; margin: 32px 0px 32px 0px;">
-                <tr><td style="font-size: 13px;">
+            <table border="0" cellpadding="0" cellspacing="0" width="590" summary="o_mail_notification" style="width:100%; margin: 32px 0px 32px 0px; display: inline-table;">
+                <tr><td style="font-size: 13px;text-align:center;">
                     <strong>Tell us how you feel about our services</strong><br/>
                     <span style="font-size: 12px; opacity: 0.5; color: #454748;">(click on one of these smileys)</span>
                 </td></tr>
                 <tr><td style="font-size: 13px;">
                     <table style="width:100%;text-align:center;margin-top:2rem;">
                         <tr>
                             <td>
-                                <a t-attf-href="/rate/{{ access_token }}/5">
+                                <a t-attf-href="/rate/{{ access_token }}/5" t-att-class="object._rating_get_operator().id and 'pe-none'">
```

Comment:
What is the purpose of that changes? 🤔 

also I think if `object._rating_get_operator().id` is falsy then class will contain `False` and I don't think it is expected.

To avoid that, we should do either:

```suggestion
                                <a t-attf-href="/rate/{{ access_token }}/5" t-att-class="object._rating_get_operator() ? 'pe-none' : ''">
```

or 

```suggestion
                                <a t-attf-href="/rate/{{ access_token }}/5" t-att-class="{'pe-none': object._rating_get_operator()}">
```

Reply:
> What is the purpose of that changes? 🤔
> 
> also I think if `object._rating_get_operator().id` is falsy then class will contain `False` and I don't think it is expected.
> 
> To avoid that, we should do either:
> 
> or

The mail template is rendered through Python, so the ternary operator will not work as expected. Therefore, we need to use it like this: `pe-none' if object._rating_get_operator1() else ''.`

Reply:
Indeed 🙂 

And yep it is good. 🙂